homeassistant/components/vallox/switch.py

```diff
@@ -0,0 +1,105 @@
+"""Support for Vallox ventilation unit switches."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from vallox_websocket_api import Vallox
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import ValloxDataUpdateCoordinator, ValloxEntity
+from .const import DOMAIN
+
+
+class ValloxSwitchEntity(ValloxEntity, SwitchEntity):
+    """Representation of a Vallox switch."""
+
+    entity_description: ValloxSwitchEntityDescription
+    _attr_entity_category: EntityCategory | None = EntityCategory.CONFIG
```

Comment:
```suggestion
    _attr_entity_category = EntityCategory.CONFIG
```

Do we need the type hint in this case?

Reply:
VS Code auto populated. I wasn't against it =)